### PR TITLE
Drupal 7 port, for great justice!

### PR DESCRIPTION
**Pull request by adam-vessey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Islandora Authority Module [![Build Status](https://travis-ci.org/discoverygarden/islandora_authority.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_authority)
```
Comment (willtp87) [inline README.md]: That last link might need to go to discoverygarden.

File: build.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+  <target name="phpcb" description="Aggregate tool output with PHP_CodeBrowser">
+    <exec executable="phpcb">
+        <arg line="--log ${basedir}/build/logs --source ${basedir} --output ${basedir}/build/code-browser"/>
+    </exec>
+  </target>
+
+  <target name="test">
+    <exec executable="drush">
+      <arg line="test-run --uri=http://localhost --xml=${basedir}/build/test 'Islandora Manuscript'" />
```
Comment (willtp87) [inline build.xml]: Islandora Manuscript?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+language: php
+php:
+  - 5.3.3
+  - 5.4
```
Comment (willtp87) [inline .travis.yml:4]: Could we add 5.5?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  - ln -s $HOME/islandora_solr_search sites/all/modules/islandora_solr_search
+  - ln -s $HOME/objective_forms sites/all/modules/objective_forms
+  - ln -s $HOME/php_lib sites/all/modules/php_lib
+  - drush en --user=1 --yes islandora_authority
+script:
+  - ant -buildfile sites/all/modules/islandora_authority/build.xml lint
+  - $ISLANDORA_DIR/tests/scripts/line_endings.sh sites/all/modules/islandora_authority
+  - drush coder-review --reviews=production,security,style,i18n,potx,sniffer islandora_authority
+  - phpcpd --names *.module,*.inc,*.test sites/all/modules/islandora_authority
+#  - drush test-run --user=1 --uri=http://localhost:8081 "Islandora Manuscript"
```
Comment (willtp87) [inline .travis.yml]: Copy pasta.

File: js/autocomplete.js [hunk 5]
```diff
@@ -128,360 +65,209 @@ Drupal.settings.islandora_authority.jsAC.prototype.select = function (node) {
...
   }
-  else {
-    //Dunno...
-    //this.select(this.selected);
+};
+
+Drupal.islandora_authority_jsAC.prototype.selectLeft = function () {
+  if (this.sub_selected) {
+    this.sub_unhighlight(this.sub_selected);
+    //this.highlight(this.selected);
```
Comment (willtp87) [inline js/autocomplete.js]: Commented out code?

File: build.xml (hunk 2 above)
Comment (adam-vessey) [inline build.xml]: Guess from where this was copied. :P

**Pull request merged**